common/src/main/java/com/viaversion/viaversion/rewriter/EntityRewriter.java

```diff
@@ -223,6 +223,17 @@ public void registerEntityDataTypeHandler(@Nullable EntityDataType itemType, @Nu
         });
     }
 
+    public void registerMinecartBlockStateHandler(final EntityType entityType) {
+        registerMinecartBlockStateHandler(entityType, 11);
+    }
```

Comment:
No 11 default, even though it didn't change for a few versions it's still too volatile of a random value

Reply:
Looking at the usages it's clearly the majority of protocols and there is no hint the index will change, but I'll remove it and rename the method then since it's not bounded to minecarts anymore?